src/app/pages/arbol-cuentas-contables/cuenta-contable/cuenta-contable.component.html

```diff
@@ -0,0 +1,152 @@
+            </mat-grid-tile>
+
+            <mat-grid-tile colspan="2" rowspan="1">
+              <section class="form-group" appearance="legacy">
+                <mat-slide-toggle [formControl]="form.controls['Nmnc']" [color]="'primary'">NMNC</mat-slide-toggle>
+              </section>
+            </mat-grid-tile>
+
+            <mat-grid-tile colspan="1" rowspan="1">
+              <section class="form-group" appearance="legacy">
+                <mat-slide-toggle [formControl]="form.controls['TieneCentroCostos']" [color]="'primary'">Tiene Centro de Costos</mat-slide-toggle>
+              </section>
+            </mat-grid-tile>
+
+            <mat-grid-tile colspan="5" rowspan="1">
+              <mat-form-field class="form-group" appearance="legacy">
+                <mat-label>Centro De costos</mat-label>
+                <mat-select [formControl]="form.controls['CentroDecostosID']">
+                  <mat-option *ngFor="let centro of centroCostos" [value]="centro.Id">{{centro.Label}}</mat-option>
+                </mat-select>
+                <mat-error *ngIf="form.controls['CentroDecostosID'].errors">
+                    {{form.controls['CentroDecostosID'].errors.msg || 'ingrese un dato valido' }}
+                </mat-error>
+              </mat-form-field>
+            </mat-grid-tile>
+
+            <mat-grid-tile colspan="1" rowspan="1">
+              <section class="form-group" appearance="legacy">
+                <mat-slide-toggle [formControl]="form.controls['RequiereBanco']" [color]="'primary'">Requiere Banco</mat-slide-toggle>
+              </section>
+            </mat-grid-tile>
+
+            <mat-grid-tile colspan="5" rowspan="1">
+              <mat-form-field class="form-group" appearance="legacy">
+                <mat-label>Banco</mat-label>
+                <input type="text" matInput placeholder="Banco" [formControl]="form.controls['Banco']">
+                <mat-error *ngIf="form.controls['Banco'].errors">
+                    {{form.controls['Banco'].errors.msg || 'ingrese un dato valido' }}
+                </mat-error>
+              </mat-form-field>              
+            </mat-grid-tile>
+
+            <mat-grid-tile colspan="6" rowspan="1">
+              <mat-form-field class="form-group" appearance="legacy">
+                <mat-label>Moneda</mat-label>
+                <mat-select [formControl]="form.controls['MonedaID']">
+                  <mat-option *ngFor="let moneda of tipoMonedas" [value]="moneda.Id">{{moneda.Label}}</mat-option>
+                </mat-select>
+                <mat-error *ngIf="form.controls['MonedaID'].errors">
+                    {{form.controls['MonedaID'].errors.msg || 'ingrese un dato valido' }}
+                </mat-error>
+              </mat-form-field>
+            </mat-grid-tile>
+          </mat-grid-list>
+          <button mat-flat-button color="primary" type="submit">Confirmar</button>
+        </form>
+      </nb-card-body>
+    </nb-card>
```

Comment:
Dejar vacía la última línea de cualquier archivo de texto plano
```suggestion
    </nb-card>

```